certs/key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Not the author, but as noted above, it's a self-signed cert, probably used for testing. It isn't used for anything in production.